src/script/src/python/pyo3/dataframe_impl.rs

```diff
@@ -223,6 +224,13 @@ impl PyDataFrame {
     }
 }
 
+#[pyfunction]
+pub(crate) fn lit(py: Python<'_>, value: PyObject) -> PyResult<PyExpr> {
```

Comment:
What does this function do?

Reply:
> What does this function do?

Convert a Python Object into a `Expr` for use in constructing literal i.e. `col("number") < lit(42)`

Reply:
Can we do this conversion automatically?

Reply:
> Can we do this conversion automatically?

Complete that in [8f95259](https://github.com/GreptimeTeam/greptimedb/pull/1122/commits/8f952599e371e646882f038267cf05fc915b5d9b)

Reply:
@discord9  After this commit, do we still need the `lit` function?

Reply:
> @discord9 After this commit, do we still need the `lit` function?

Could still be used in some scenarios I think? For example a single Literal `lit(3)` in some case?